src/lib/CollisionPrevention/CollisionPrevention.hpp

```diff
@@ -72,13 +72,12 @@ class CollisionPrevention : public ModuleParams
 
 private:
 
-	bool _is_active = true;
 	bool _interfering = false;		/**< states if the collision prevention interferes with the user input */
 
-	orb_advert_t _constraints_pub{nullptr};		/**< constraints publication */
+	orb_advert_t _constraints_pub = nullptr;		/**< constraints publication */
```

Comment:
This change is the wrong way around because `var = val` does two things: 1) create a non-initialized chunk of memory for var and 2) create an initialized chunk of memory and copy it over to var. Use the curly braces for all these so that the chunk of memory gets created initialized.

Reply:
Hah I thought so. Thanks for the explanation, I'll fix it.

Reply:
Done